app/src/main/java/com/simplecity/amp_library/ui/views/CustomColorPicker.java

```diff
@@ -19,8 +18,6 @@
         implements SeekBar.OnSeekBarChangeListener,
         TextWatcher {
 
-    private static final String TAG = "ColorPicker";
```

Comment:
I'd prefer to leave TAGs in place. Makes it simpler in future to write log statements if the tag is still around.